lamindb/models/query_set.py

```diff
@@ -689,9 +691,9 @@ def first(self) -> Record | None:
             return None
         return self[0]
 
-    def one(self) -> Record:
+    def one(self, does_not_exist_msg: str | None = None) -> Record:
```

Comment:
Rather than exposing this message as an argument to the user I suggest you replace the call of `.one()` up there with a call to the non-user facing `one_helper()`.

`.one()` is the user-facing API that we should keep free of internal logic! ☺️